pkg/volume/volume.go

```diff
@@ -65,6 +66,12 @@ type Recycler interface {
 	Recycle() error
 }
 
+// Create adds a new resource in the storage provider and creates a PersistentVolume for the new resource.
+// Calls to Create should block until complete.
+type Creater interface {
```

Comment:
Shouldn't this be `Creator`?  I don't want to pick a nit at the last minute if this is otherwise good.  Follow-up if you agree?